Exercise_1.py

```diff
@@ -1,22 +1,37 @@
+  
+#Time Complexity : O(log n)
+#Space Complexity : O(1)
+#Did this code successfully run on Leetcode : yes
+#Any problem you faced while coding this : No
+
 # Python code to implement iterative Binary  
 # Search. 
   
 # It returns location of x in given array arr  
 # if present, else returns -1 
-def binarySearch(arr, l, r, x): 
+def binarySearch(arr, l, r, x):
+  while l < r:
+    mid = (l + r) //2
+    if arr[mid] == x:
+      return mid
+    elif arr[mid] < x:
+      l += 1
```

Comment:
You can make l = mid+1 and r = mid-1; Just keep this in mind and you'll learn the logic soon in your class

Reply:
Oh yes, got it. Thanks.